apps/SageAccountWeb/src/components/ProfileValidation/TermsAndConditionsSignature.tsx

```diff
@@ -0,0 +1,49 @@
+import { TextField } from '@mui/material'
+import { StyledFormControl } from 'components/StyledComponents'
+import React from 'react'
+import { Typography } from 'synapse-react-client'
```

Comment:
Would prefer to import `Typography` from MUI instead of SRC but if that ends up breaking things, don't worry about it for now. I suspect there are a few problems throughout the app related to how we're applying the theme

Reply:
I'll update the import -- it doesn't cause any noticeable differences here. It does look like Typography is imported from synapse-react-client in several other components, so may be something to clean up in the future